### PR TITLE
Using Big Data analytics to fight cybersecurity threats

### LINKED ISSUE #6575: Using big data analytics to fight cybersecurity threats
### Topic Suggestion
Fighting cybersecurity threats with big data

#### Pre-submission advice
There are three main challenges businesses are running into with big data protecting sensitive and personal information, Data rights and ownership, and not analyzing the data. But what are the threats and opportunities involved in Big data and cybersecurity?

Big Data is generally defined as being dense in variety, velocity, and volume. Big data has ushered in new analytics and security solutions to protect data and prevent future cyberattacks from a cyber security standpoint.
But just as big data has opened up new possibilities for cyber security teams, it has also allowed cybercriminals to access mass quantities of sensitive and personal information through the use of advanced technologies.

Cybersecurity analytics: Increasingly sophisticated attack methods used by cybercriminals and the growing role of malicious insiders in several recent large-scale security breaches indicate that traditional approaches to information security can no longer keep up. So, where does security analytics fits in? 

#### Key takeaways
After participating in this tutorial, you should fully understand
- What Big Data and Cybersecurity is and its purpose.
- Understand major points where cybersecurity analytics fits in

#### Article quality
After this lesson, you will develop an insight into the level of awareness and current approaches in information security fraud detection in organizations worldwide. Unlike some other blogs relating to security analytics, I will closely work with you and show you in-depth measures and plans of significant data security analytics initiatives across different sectors.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquote notes as you fill in the form before you submit.
We look forward to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Petekavinsky**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6575

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by jerimkaura** [comment]
Hello @Petekavinsky 

Thanks for submitting your article for review. You must have removed the pre-submission checks. I am adding them for you so that you do the checks before we begin the review.

**Comment by Petekavinsky** [comment]
@jerimkaura, Thank you I have checked the boxes

File: content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+### Outputs
+Capturing, gathering, storing, and processing data is not straightforward or inexpensive, and it is useless until the data is meaningful; it must also be accessible when needed.
+Three major enablers are present:
+- Mobile Information may now be distributed in real-time thanks to established mobile networks.
+- The ability to review big and complicated data sets has become accessible to the ordinary business user because of visual/interactive technologies.
+- Human resources There is a new breed of team members that can handle the complexity of big data and reduce the output for everyday use.
+
+### Cyber security and Big Data analytics
+1. In Fraud Detection, Big Data Analytics is Used. Statistical approaches and artificial intelligence techniques are the two main fraud detection tools.
```
Comment (jerimkaura) [inline content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md:71]: At least introduce the reader to what you are talking about in this context.

File: content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+7. Anomalies in transaction or user behaviour relative to previously known models and profiles are detected using matching algorithms. Techniques to eliminate false alerts, quantify risks, and anticipate the future of existing transactions or users are also required. Fraud management requires a great deal of knowledge.
+
+The main AI techniques used for fraud management include [AI]:
+1. Use data mining to classify automatically, cluster, and segment data and uncover relationships and rules that may indicate exciting trends, such as fraud tendencies.
+2. Expert systems that encode knowledge in the form of rules for identifying fraud.
+3. Pattern recognition to find approximations of classes, clusters, or patterns of suspicious behaviour either automatically (unsupervised) or by matching given inputs.
+4. Automated fraud detection using machine learning approaches.
+5. Neural networks can learn suspicious patterns from samples and then utilize them to detect them in the future.
+
+### Anamoly-Based Intrusion Detected Using Big Data Analytics
```
Comment (jerimkaura) [inline content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md:88]: This heading seems unclear.

File: content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Anomaly detection algorithms are relatively easy to set up and operate independently. Some key performance indicators are selected for each event, and thresholds are established. The occurrence is flagged for further examination when a certain threshold is reached. The choice of indicators to be monitored, the analysis time, and the threshold value choices all impact the success of this strategy.
+
+Anomaly detection algorithms are easy to set up and operate without any human interaction. However, the parameters monitored, the analysis time, and the threshold value settings all impact the success of this strategy.
+
+### Assist in the gathering of security information
+They can speed up the process of forensic data correlation and create proactive security responses.
```
Comment (jerimkaura) [inline content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md:94]: What can assist in the gathering of ......?

File: content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+### Considerations for extensive data security
+1. Collaborating with peers in the sector to develop industry standards, avoid government requirements, and share best practices
+2. To safeguard sensitive information supplied by third parties, attribute-based encryption is used.
+3. Hadoop, for example, is secure open-source software.
+4. Keep and monitor audit logs for all aspects of the firm.
+Overall, big data has significant opportunities for enterprises that extend far beyond improved business intelligence.
+ Big data has the potential to improve cyber security. However, to capitalize on the numerous opportunities presented by big data, businesses must accept the duty and risk of data protection.
+ 
+### Challenges
```
Comment (jerimkaura) [inline content/articles/using-big-data-analytics-to-fight-cyber-threats/index.md:107]: Use descriptive heading for easy understanding and effective SEO

**Review by jerimkaura — CHANGES_REQUESTED** [review]
Hello @Petekavinsky 

Thanks for submitting your article for review. You have a wonderful article but considering the major readability issues identified, consider re-reading the article to eliminate the mistakes and polish it well.

We highly encourage our author to only create PRs for publish-ready articles.

**Comment by Petekavinsky** [comment]
@jerimkaura,I've completed all of the necessary changes.

**Comment by Adespscientist** [comment]
@jerimkaura, What is the update on my PR please

**Comment by WanjaMIKE** [comment]
Hi @Petekavinsky 

Your article appears to have many sentences that are taken verbatim from third-party sources. We can't publish any content that is not original or properly cited/referenced. There are many free online tools where you can check this (such as quetext.com, and if you're uncertain about the definitions around plagiarism, this is a [good resource to reference](https://www.scribbr.com/category/plagiarism/). We're really looking for unique/original content.

[New Scan 5_20 PM Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/8126507/New.Scan.5_20.PM.Copyleaks.Report.pdf)

**Pull request closed without merging**